extlibs/headers/glad/include/glad/wgl.h

```diff
@@ -571,16 +567,6 @@ static int gladLoadWGLUserPtr(HDC hdc, GLADuserptrloadfunc load, void *userptr)
 static int gladLoadWGL(HDC hdc, GLADloadfunc load) {
     return gladLoadWGLUserPtr(hdc, sf_glad_wgl_get_proc_from_userptr, GLAD_GNUC_EXTENSION (void*) load);
 }
- 
-
-#ifdef SF_GLAD_WGL
-
-static int gladLoaderLoadWGL(HDC hdc) {
-    return gladLoadWGLUserPtr(hdc, sf_glad_wgl_get_proc_from_userptr, GLAD_GNUC_EXTENSION (void*) wglGetProcAddress);
-}
-
-
-#endif /* SF_GLAD_WGL */
```

Comment:
Were these changes required, @SuperV1234 and can/should we keep them, @binary1248?

Reply:
I removed the functions in my first commit. They are not used and would cause warnings on some compilers.